mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxPath.java

```diff
@@ -48,6 +52,10 @@ public static MailboxPath forUser(String username, String mailboxName) {
         return new MailboxPath(MailboxConstants.USER_NAMESPACE, username, mailboxName);
     }
 
+    private static final String INVALID_CHARS = "%*&#";
+    private static final CharMatcher INVALID_CHARS_MATCHER = CharMatcher.anyOf(INVALID_CHARS);
+    public  static final int MAX_MAILBOX_NAME_LENGTH = 200;
```

Comment:
why? In fact when we are adding such value, I think we should add a reference to the authoritative RFC.

Reply:
> why? In fact when we are adding such value, I think we should add a reference to the authoritative RFC.

It's not about RFC, it's some JPA string field size at the origin of this limit.

Past code, past decision, just refactored. Is it worth getting this debate?

Reply:
no debate, but then just a little comment explaining where this value comes from would be great.